Makefile

```diff
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
+INSTRUCLAB_VERSION ?=
```

Comment:
misspelled "INSTRUCLAB" here, but it has the 2nd T when used below.

Reply:
Thanks